bin/todo

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
```

Comment:
Since this is mostly for developers, it's up to them to patch this for any other python names. Keep in mind that setup.py also uses this hashbang.

We might want to document this detail though.